### ISSUE TITLE
✅ 🔗 🐞 Cursor is not pointer in Taxonomy Filter element (Duplicate of #24322) - [ED-14522]

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#25448]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi,

In the Taxonomy Filter element, if we move the mouse over the available options, the cursor is not a pointer.

### Steps to reproduce

On the desktop, move the mouse over the items in the Taxonomy Filter element. It is clear that the cursor is not a pointer.

### Expected behavior

-

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.54 (Win64) OpenSSL/1.1.1q PHP/8.1.10
	MySQL version: MySQL Community Server - GPL v8.0.30
	PHP Version: 8.1.10
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 2G
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.3
	Site URL: http://test-website.test
	Home URL: http://test-website.test
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.9.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.20.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.20.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved Asset Loading: Active
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active
	Build with AI: Inactive
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Active
	Optimize Image Loading: Active
	Pages Panel: Inactive by default
	Notes: Inactive
	Display Conditions: Active
	Form Submissions: Active
	Menu: Active
	Taxonomy Filter: Active


== Log ==
	
JS: showing 13 of 13JS: 2024-01-08 20:33:47 [error X 1145][../wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:14799] Cannot read properties of null (reading 'clone') 
JS: 2024-01-08 20:33:47 [error X 31][../wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:19552] Cannot read properties of null (reading 'changestop') 
JS: 2024-01-09 09:14:20 [error X 1][../wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.18.1:2:64171] Cannot read properties of undefined (reading 'createTemplate') 
JS: 2024-01-09 09:19:50 [error X 2][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.18.2:3:954553] Cannot read properties of undefined (reading 'settings') 
JS: 2024-01-09 21:17:49 [error X 108][../wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.18.2:3:7470] Cannot read properties of null (reading 'editor') 
JS: 2024-01-24 23:44:23 [error X 6][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.18.2:3:806906] Cannot read properties of undefined (reading 'localized') 
JS: 2024-02-16 12:39:51 [error X 1][../wp-content/plugins/elementor/assets/js/web-cli.min.js?ver=3.19.2:3:82947] Data commands: `` not found. 
JS: 2024-02-17 12:03:39 [error X 2][../wp-admin/load-scripts.php?c=1&loadchunk_0=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=6.4.3:2:28760] Cannot read properties of undefined (reading 'value') 
JS: 2024-03-06 20:17:07 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1442") has already been destroyed and cannot be used. 
JS: 2024-03-06 20:28:04 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view1446") has already been destroyed and cannot be used. 
JS: 2024-03-07 19:37:10 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view7226") has already been destroyed and cannot be used. 
JS: 2024-03-07 19:47:56 [error X 1][../wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: "view8697") has already been destroyed and cannot be used. 
JS: 2024-03-07 20:04:20 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.2:3:814165] Cannot read properties of undefined (reading 'toLowerCase') 

PHP: showing 4 of 4PHP: 2024-01-08 20:36:29 [warning X 1][..\wp-content\plugins\elementor\core\logger\items\js.php::17] Undefined array key "url" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-02-14 22:20:16 [error X 1][..\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php::200] Uncaught Error: Class "\ElementorPro\Modules\ThemeBuilder\Conditions\General" not found in ..\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php:200
Stack trace:
#0 ..\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php(282): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->register_condition('general')
#1 ..\wp-includes\class-wp-hook.php(324): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->register_conditions('')
#2 ..\wp-includes\class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#3 ..\wp-includes\plugin.php(517): WP_Hook->do_action(Array)
#4 ..\wp-settings.php(665): do_action('wp_loaded')
#5 ..\wp-config.php(103): require_once('C:\\laragon\\www\\...')
#6 ..\wp-load.php(50): require_once('C:\\laragon\\www\\...')
#7 ..\wp-admin\admin-ajax.php(22): require_once('C:\\laragon\\www\\...')
#8 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-06 20:07:33 [notice X 75][..\wp-content\plugins\elementor-pro\modules\custom-css\module.php::96] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-11 23:24:11 [notice X 7][..\wp-content\plugins\elementor\includes\base\controls-stack.php::228] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 252024-02-14 22:22:17 [info] elementor-pro::elementor_pro_updater Started 
2024-02-14 22:22:17 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-02-14 22:22:17 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-02-14 22:22:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.1',
  'to' => '3.19.2',
)]
2024-03-11 23:19:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.2',
  'to' => '3.20.0',
)]
2024-03-11 23:19:19 [info] elementor-pro::elementor_pro_updater Started 
2024-03-11 23:19:19 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-03-11 23:19:19 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-03-11 23:19:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.2',
  'to' => '3.20.0',
)]
2024-03-11 23:21:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.2',
  'to' => '3.20.0',
)]
2024-03-11 23:21:20 [info] elementor::elementor_updater Started 
2024-03-11 23:21:20 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-11 23:21:20 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-11 23:21:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.2',
  'to' => '3.20.0',
)]
2024-03-15 23:06:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]
2024-03-15 23:06:14 [info] elementor::elementor_updater Started 
2024-03-15 23:06:14 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-15 23:06:14 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-15 23:06:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]
2024-03-15 23:06:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
Same goes for the mega menu widget. If I remember correctly, Nicholas said in a beta test thread that it is normal. (Not that I also think that)

**Comment by nicholaszein** [comment]
Hi @79ho3ein! 

Thank you for submitting your issue! 🙏 

I can confirm this is an issue with the Taxonomy Filter widget. It's caused by a CSS rule used to unset all styles in the buttons of this widget to avoid inheriting button styles applied by your theme or somewhere else. Unfortunately, there is no declaration to apply the `cursor: pointer` again like we have in the Mega Menu.

🏷️I've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. 

As a workaround, for now you can add the following declaration to your Taxonomy Filter widget:

```css
selector .elementor-widget-taxonomy-filter .e-filter-item {
    cursor: pointer;
}
```

Thanks for understanding!

Best regards

----

PS: @toineenzo, for the Mega Menu, when a menu item doesn't have a link, it shouldn't change the cursor to a pointer. This is by design, and follows accessibility best practices. If the item does have a link, then the pointer will appear.

**Comment by nicholaszein** [comment]
Hey @79ho3ein and @toineenzo!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
